### PR TITLE
Updating dashboard node config (BSC#1166052)

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: xml/admin_about.xml [hunk 1]
```diff
@@ -266,7 +263,10 @@
        The &ceph; manager (MGR) collects the state information from the whole
```
Comment (lproven) [inline xml/admin_about.xml]: I think there is an entity for this? &mgr;

Comment (tbazant) [reply xml/admin_about.xml]: +1

File: xml/admin_about.xml [hunk 2]
```diff
@@ -266,7 +263,10 @@
...
        cluster. The &ceph; manager daemon runs alongside the monitor daemons.
        It provides additional monitoring, and interfaces the external
-       monitoring and management systems.
+       monitoring and management systems. It includes other services
+       as well, for example the &dashboard; Web UI with the
+       <emphasis>&grafana;</emphasis> dashboard backed by the
```
Comment (LenzGr) [inline xml/admin_about.xml]: Actually, both Prometheus and Grafana are independent instances that do not have any direct relation to the Dashboard or the Ceph Mgr daemon. The Dashboard is capable of "consuming"/embedding Grafana dashboards, which in turn uses Prometheus to collect the metrics that it visualizes.

Comment (asettle) [reply xml/admin_about.xml]: I think it's best to just remove everything after "web UI" in that case :) thanks Lenz

**Pull request merged**